src/components/ui/searchBar.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
```

Comment:
if we are going to use camel case for the name, please change split_view to splitView so they have the same style